sharding/client/client.go

```diff
@@ -1,6 +1,4 @@
-package sharding
-
-//go:generate abigen --sol contracts/sharding_manager.sol --pkg contracts --out contracts/sharding_manager.go
+package client
```

Comment:
Let's please rename this to sharding_client.go. client is ambiguous with the ethclient that is a part of the struct.

Reply:
This isn't related to the filename, but the package path. Do we need the redundancy of sharding/sharding_client/client.go?

Reply:
Let's revisit this in the future. I'm going to leave for now.